backends/calico/client.go

```diff
@@ -342,6 +347,24 @@ func (c *client) keyUpdated(key string) {
 		if rev != c.cacheRevision && strings.HasPrefix(key, prefix) {
 			log.Debug("Updating prefix to rev %d", c.cacheRevision)
 			c.revisionsByPrefix[prefix] = c.cacheRevision
+
+			// If this is a change to either the global log level, or the per-node
+			// log level, then configure confd's log level to match.
+			if strings.HasSuffix(key, "loglevel") {
```

Comment:
Need a better way - probably should just check the exact keys.

Reply:
Yeah they should be:
```
/calico/bgp/v1/global/loglevel
/calico/bgp/v1/host/<host>/loglevel
```

Reply:
oh actually you already have those keys defined.

Reply:
yeah - you already have those keys defined so maybe it's worth tightening up a bit

Reply:
but tbh - i don't think it matters too much since the updateLogLevel would just no-op if it wasn't the correct log level.